spec/models/timetables_spec.rb

```diff
@@ -110,6 +112,8 @@
         open_time: "09:00",
         close_time: "17:00",
         note: "",
+        short_note: "This is a short note",
+				short_note_url: "https://library.columbia.edu",
```

Comment:
alignment/tab mismatch. Same comment for line 103 above.

Reply:
Good catch, thanks!  Looked fine in my IDE, despite the space/tab difference, but I just indented and de-dented the entire file and that fixed it.  Unfortunately, rubocop isn't being used on this repo right now, otherwise that might have been caught automatically.